mtbf_driver/tests/mtbf/settings/test_mtbf_settings_bt.py

```diff
@@ -2,32 +2,37 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import time
 from mtbf_driver.MtbfTestCase import GaiaMtbfTestCase
-from mtbf_driver.mtbf_apps.settings.app import MTBF_Settings
-
+from mtbf_driver.mtbf_apps.settings.app import MTBF_Settings as Settings
 
 class TestSettingsBluetooth(GaiaMtbfTestCase):
 
     def setUp(self):
         GaiaMtbfTestCase.setUp(self)
-        self.settings = MTBF_Settings(self.marionette)
-        self.settings.launch()
 
     def test_turn_on_and_off_bt(self):
-        self.settings.go_back()
+        device_name = str(time.time())
```

Comment:
Recommend to use uuid here https://docs.python.org/2/library/uuid.html
Although timestamp is rational unique but less readable here.